### ISSUE TITLE
Cinnamon: can't turn off screen.

### ISSUE THREAD

**Issue by nuusers** [NixOS/nixpkgs#192235]
### Describe the bug
I set turn off the screen when incative fot 5 minutes.
When I am not active, after 5 minutes, the screen will close once and then open instantly.
Use xset can reproduce.
### Steps To Reproduce
```console
xset dpms force off
```

### Expected behavior
Screen will close once and then open instantly.

### Notify maintainers

@bobby285271 
### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.68, NixOS, 22.11 (Raccoon), 22.11.20220921.7537ce3`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by bobby285271** [comment]
Does `sleep 2 && xset dpms force off` also reproduce the issue for you?

**Comment by nuusers** [comment]
@bobby285271 Yes, and try pantheon de reproduce too, but when use wm(dwm) just startx can’t reproduce, so maybe lightdm bug?
